python/artm/regularizers.py

```diff
@@ -66,29 +67,63 @@ def _update_config(self, obj, first=False):
 
 
 class Regularizers(object):
-    def __init__(self, master):
+    def __init__(self, master, force_add=False):
         self._data = {}
         self._master = master
+        self._force_add = force_add
 
     def add(self, regularizer):
-        if regularizer.name not in self._data:
-            self._master.create_regularizer(regularizer.name, regularizer.config, regularizer.tau, regularizer.gamma)
-            regularizer._master = self._master
-            self._data[regularizer.name] = regularizer
+        name = regularizer.name
+        if name in self._data and not self._force_add:
+            raise AttributeError("Unable to replace existing regularizer with force_add option turned off.\
+If you really want to replace existing regularizer, set force_add option to True")
+        # next statement represents ternary operator
+        # in C/C++:  `res = cond ? x : y`
+        # in Python: `res = cond and x or y` or
+        #            `res = cond and x or not cond and y` (when x may be equivalent to False)
+        register_func = (name not in self._data and self._master.create_regularizer or
```

Comment:
Why not `x if cond else y`? I though this is the standard way to represent trenary operator in python:
http://stackoverflow.com/questions/394809/does-python-have-a-ternary-conditional-operator